doc/classes/HScrollBar.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="HScrollBar" inherits="ScrollBar" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Horizontal scroll bar.
+		A horizontal scrollbar that goes from left (min) to right (max).
 	</brief_description>
 	<description>
-		Horizontal version of [ScrollBar], which goes from left (min) to right (max).
+		A horizontal scrollbar that goes from left (min) to right (max).
```

Comment:
The old description was fine.

Reply:
Meh. Having to repeat myself is annoying, but no, it's not a horizontal version. ScrollBar is an abstract node.

Reply:
> It's also not just a horizontal ScrollBar, as that class is abstract.

It is a ScrollBar, or more accurately, an implementation of that class. It has no members other than the theme items.

Reply:
Well, yeah, the "Inherits: [Class]" line is always there. So it's not "A horizontal [ScrollBar]" but more like "A horizontal version of [ScrollBar] that can actually be added to the scene tree". 🤔 

I'll go over the H/V variations and brainstorm some more

Reply:
Scrollbar: Abstract base class for scrollbars, used to navigate through content that is too large to fit within the visible area of a control. Scrollbars are [Range]-based controls.

HScrollBar and VScrollBar are the same, but instead saying "content that extends beyond the visible height/width"

Reply:
You could say `typically used`, to make it clear that a ScrollBar alone does not offer any scrolling (and that it can be used for something else too).

Reply:
Slider: Abstract base class for sliders, used to adjust a value by moving a marker along a horizontal or vertical axis. Sliders are [Range]-based controls.

Same variations as above for the variants.

Reply:
The "marker" is called grabber.